### ISSUE TITLE
Floating Chests

### ISSUE THREAD

**Issue by DebugingLonks** [Wynncraft/Issues#7154]
I have noticed recently there are lots of floating chests in wynn. I even found some secret ones...
Are chests meant to be so high that only a archer can reach them, and if you opened you can't even get anything because you fall xD (I mean to get close enough to the chest you have to be ontop of it. The chest is over nothing, so you are too far away from chest and it closes) some examples are:

- Ancient Nemract has a few (look straight up from the cave that leads to dungeon you should see 2)
- Wynnter fair (No one noticed this I think) I'd have to show you, its like 30 blocks in the air and its a tier 4.
- There is one in troms (idk where I just remember it exists)

If this is not a bug/glitch, comment it and I will close issue.

**Comment by LordHelmchen98** [comment]
No Im pretty sure this is not how its supposed to be. I guess if you can get some coords a builder can most likely fix them.

**Comment by Imaxe** [comment]
Message me or any other GM the coords and we'll fix it :)

**Comment by DebugMC** [comment]
kk

**Comment by Moe-Ronickah** [comment]
"Ancient Nemract has a few (look straight up from the cave that leads to dungeon you should see 2)"
  *Don't remove those.  They are intentional and can be reached by simply jumping.*

"Wynnter fair (No one noticed this I think) I'd have to show you, its like 30 blocks in the air and its a tier 4."
    *It was in the Xmas tree, and noticed by plenty of players, however, it spawns in by different means than builds use.  This can be removed.*

**Comment by DebugMC** [comment]
No, its not that one :P go to where the ice game used to be, there's this circular wooden structure +look

**Comment by DebugMC** [comment]
@XavierEXE can tell you where it is since I don't have coords btw

**Comment by Moe-Ronickah** [comment]
@DebugMC , can you give me the coords of the one the zombie dungeon, please?  I can't find it.

**Comment by Celsonic** [comment]
Found one near galleon's graveyard a while back. Let me see if I can find it again.

**Comment by Moe-Ronickah** [comment]
@DebugMC , I'm still waiting for those coordinates.

**Comment by DebugMC** [comment]
Sorry about that, im on a trip for the rest of the week ill check when I am
back.
